docs/advanced/pcidevices.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 0
+sidebar_label: PCI Devices
+title: ""
+---
+
+# PCI Devices
+
+A PCIDevice represents a host device with a PCI address. 
```

Comment:
```suggestion
A PCI device represents a host device with a PCI address. 
```

Reply:
But PCIDevice is the name of the CRD, it's a literal